pkg/workspace/library_module.go

```diff
@@ -76,10 +76,10 @@ func (b LibraryModule) Get(thread *starlark.Thread, f *starlark.Builtin,
 	if err != nil {
 		return starlark.None, err
 	}
+	rootLibrary := thread.Local(threadRootLibraryKey).(*Library)
+	libraryCtx := LibraryExecutionContext{Current: foundLib, Root: rootLibrary}
```

Comment:
This — as is — is a breaking change.

Any existing `@load("/some-file")` currently loads from the root _of the current library_;
with this change would load from _the root library_.

Not acceptable.